core/lib/types/src/transaction_request.rs

```diff
@@ -1543,4 +1545,33 @@ mod tests {
         let tx_request = TransactionRequest::from(call_request.clone());
         assert_eq!(tx_request.input, call_request.input.unwrap());
     }
+
+    #[test]
+    fn test_eip712_without_field_to() {
+        let factory_dep = vec![2u8; 1600000];
+        let random_tx_max_size = 100_000; // bytes
+        let eip712_call_request = CallRequest {
+            from: Some(Address::random()),
+            to: None,
+            gas: Some(U256::from(12u32)),
+            gas_price: Some(U256::from(12u32)),
+            max_fee_per_gas: Some(U256::from(12u32)),
+            max_priority_fee_per_gas: Some(U256::from(12u32)),
+            value: Some(U256::from(12u32)),
+            data: Some(Bytes(factory_dep)),
```

Comment:
Dumb question: Why is the var called `factory_dep`? It's calldata for the transaction, right? AFAIU, factory deps [are taken](https://github.com/matter-labs/zksync-era/blob/d287725778b8dc625ed74088ea5dd2a8982d0224/core/lib/types/src/transaction_request.rs#L826) from `eip712_meta`.

Reply:
Some ancient artifact I guess, just copypasted it. Will rename